.gitlab-ci.yml

```diff
@@ -30,6 +30,7 @@ stages:
 - functional_tests
 
 before_script:
+  - export PROJECT_ROOT=`pwd`
```

Comment:
this should be handled by `source environment `

Reply:
true! Fixing this.